etc/tendrl/node-agent/node-agent-dev.conf.yaml

```diff
@@ -4,3 +4,8 @@ etcd_connection: 0.0.0.0
 log_cfg_path: /etc/tendrl/node-agent/node-agent_logging.yaml
 log_level: DEBUG
 package_source_type: pip
+logging_socket_path: /var/run/tendrl/message.sock
+ceph_provisioner: CephInstallerPlugin
```

Comment:
This needs to be hardcoded in tendrl.objects.definitions.master (.yml) , There is no reason strong enough to let end users understand theses internals and handle them via config